### ISSUE TITLE
WAD files

### ISSUE THREAD

**Issue by camigord** [pathak22/noreward-rl#5]
Hi @pathak22,

very interesting paper. I have been trying to replicate your results in pytorch using Vizdoom. You mention that by default your code runs the "dense-reward" setup, with only 15 possible starting locations. In the original my_way_home.wad file provided by Vizdoom, however, all the 17 locations are available. 

Could you please explain me where could I get the "dense-reward" wad file and the 'sparse' wad file to compare? I would like to avoid having to install doom-builder and modify the files...

Thanks,

**Comment by camigord** [comment]
sorry. found the files.

**Comment by pathak22** [comment]
@camigord I have also released the full training code now with a detailed description for setting up VizDoom wads and environments. Please see the latest commit. VizDoom description here: https://github.com/pathak22/noreward-rl/tree/master/doomFiles
